addons/web_editor/static/lib/odoo-editor/src/commands/align.js

```diff
@@ -1,17 +1,16 @@
 /** @odoo-module **/
 import { childNodeIndex, isBlock } from '../utils/utils.js';
 
-Text.prototype.oAlign = function (offset, mode) {
```

Comment:
I've been quite confused by this argument for a few time. It's never used and that's why you removed it, right ?